dts/bindings/reserved-memory/reserved-memory.yaml

```diff
@@ -0,0 +1,26 @@
+description: Reserved memory
+
+compatible: "reserved-memory"
+
+properties:
+    "#address-cells":
```

Comment:
Please get this and `#size-cells` from base.yaml like this:

```
include:
  - name: base.yaml
    property-allowlist: ['#address-cells', '#size-cells']
```

Reply:
done.